packages/kbn-dev-utils/src/kbn_client/kbn_client_requester.ts

```diff
@@ -60,6 +60,7 @@ export interface ReqOptions {
   path: string;
   query?: Record<string, any>;
   method: 'GET' | 'POST' | 'PUT' | 'DELETE';
+  headers?: Record<string, string>;
```

Comment:
I think we can revert the changes in this file. Originally, I was trying to use this client to make Kibana API calls using an API key, which requires us to pass in `headers`, but that did not seem to work because the userid/password were still being used in the URL for the request.

Reply:
For the Kibana Team:

It would be great if `KbnClient` allows us to make `.request()` calls using API keys - see the sub-class created here in this PR under this file: https://github.com/elastic/kibana/pull/74305/files#diff-52d0282fedafcd94a4ef5a5021c90c80R22-R38